tests/test_ntp_server.rb

```diff
@@ -83,4 +92,54 @@ def test_multiple
     refute_includes(Cisco::NtpServer.ntpservers, id1)
     refute_includes(Cisco::NtpServer.ntpservers, id2)
   end
+
+  def test_domain_name
```

Comment:
It does, but requires DNS to be functional in the default VRF.

On some platforms / os the device will silently fail if it cannot resolve the hostname.

I'm actually okay with taking out the hostname test as it's problematic.